Systematics/python/flashggDiPhotonSystematics2018_cfi.py

```diff
@@ -5,312 +5,488 @@
 photonScaleUncertBins = cms.PSet()
```

Comment:
It looks like this files was edited by accident. In order to avoid injecting a large, unnecessary, diff into the history please revert back the changes and only commit the relevant ones.